scripts/uberenv/packages/axom/package.py

```diff
@@ -55,7 +55,7 @@ class Axom(CMakePackage, CudaPackage):
     version('develop', branch='develop', submodules=True)
 
     # SERAC EDIT START
-    version('0.4.0p1', commit='d74c5d971c62a2263efa438736440ff28620dc3d', submodules="True")
```

Comment:
Are you sure this caused axom to get incremented?  I don't know if they track it in the mirror based on version number or commit...

Reply:
I was worried about that too, but it must have worked because the new Inlet features compile and run on Azure.

Reply:
Maybe in the future we should rename these to like 0.4.0serac so it implies its a moving target specific to us.